sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/view/StringPredicates.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.view;
+
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+
+/** Re-usable string predicates. */
+public final class StringPredicates {
+  private StringPredicates() {}
+
+  /** A string predicaate that matches all strings. */
+  public static final Predicate<String> ALL = value -> true;
+
+  /** A string predicate that does exact string matching. */
+  public static Predicate<String> exact(String match) {
+    return input -> (input != null) && match.equals(input);
```

Comment:
Intellij tells me the `(input != null)` check is unnecessary before `match.equals(`.

Reply:
removed